tests/test_auth.py

```diff
@@ -96,7 +96,6 @@ def create_auth(requests_mock: req_mock.Mocker):
 class TestAuth:
```

Comment:
There's more to clean up here, workspace endpoint mocking is not needed any more